### PR TITLE
Set file local variables for consistent code style

### PR DESCRIPTION
**Pull request by snogge**

### CODE REVIEW

File: buttercup.el [hunk 1]
```diff
@@ -1701,3 +1701,8 @@ With buttercup minor mode active the following is activated:
 
 (provide 'buttercup)
 ;;; buttercup.el ends here
```
Comment (jorgenschaefer) [inline buttercup.el]: This comment should mean that the file ends here, and not that it is then followed by further comments.

Comment (snogge) [reply buttercup.el]: Updated.

File: buttercup.el [hunk 2]
```diff
@@ -1701,5 +1701,10 @@ With buttercup minor mode active the following is activated:
       (cl-dolist (form imenu-forms)
         (setq imenu-generic-expression (delete form imenu-generic-expression))))))
 
+;; Local Variables:
+;; indent-tabs-mode: nil
+;; sentence-end-double-space: nil
+;; End:
 (provide 'buttercup)
 ;;; buttercup.el ends here
+
```
Comment (jorgenschaefer) [inline buttercup.el]: At the risk of sounding highly pedantic, but there should not be an empty line at the end of the file, either :-D

Comment (snogge) [reply buttercup.el]: I have no problem complying with that, but I am a bit surprised. Where is that policy stated?

Comment (snogge) [reply buttercup.el]: Sorry, I see what you mean now. I'll push an update.

Comment (snogge) [reply buttercup.el]: Updated now. Rebased the branch as well.

**Comment by jorgenschaefer** [comment]
Thank you!

**Pull request merged**